gcloud/orb.yaml

```diff
@@ -457,7 +462,14 @@ jobs:
           project: <<parameters.project>>
       - configure-docker:
           docker_layer_caching: <<parameters.docker_layer_caching>>
-      - checkout
+      - when:
+          condition: <<parameters.custom-checkout>>
+          steps:
+            - run: git clone <<parameters.custom-checkout>>
```

Comment:
I recommend using `git clone --depth=1` here, since we don't need to pull the full repo history as far as I can see.